artemis-server-osgi/src/main/java/org/apache/activemq/artemis/osgi/OsgiBroker.java

```diff
@@ -1,223 +1,252 @@
-    * simple at the mo as e only have core and jms but will need impproving if
-    * we get more.
-    */
-   public ArrayList<ActiveMQComponent> getComponentsByStartOrder(Map<String, ActiveMQComponent> components) {
-      ArrayList<ActiveMQComponent> activeMQComponents = new ArrayList<>();
-      ActiveMQComponent jmsComponent = components.get("jms");
-      if (jmsComponent != null) {
-         activeMQComponents.add(jmsComponent);
-      }
-      activeMQComponents.add(components.get("core"));
-      return activeMQComponents;
-   }
-
-   public void register(BundleContext context, Dictionary<String, ?> properties) {
-      registrations = new HashMap<>();
-      for (Map.Entry<String, ActiveMQComponent> component : getComponents().entrySet()) {
-         String[] classes = getInterfaces(component.getValue());
-         Hashtable<String, Object> props = new Hashtable<>();
-         for (Enumeration<String> keyEnum = properties.keys(); keyEnum.hasMoreElements(); ) {
-            String key = keyEnum.nextElement();
-            Object val = properties.get(key);
-            props.put(key, val);
-         }
-         ServiceRegistration<?> registration = context.registerService(classes, component.getValue(), props);
-         registrations.put(component.getKey(), registration);
-      }
-   }
-
-   private String[] getInterfaces(ActiveMQComponent value) {
-      Set<String> interfaces = new HashSet<>();
-      getInterfaces(value.getClass(), interfaces);
-      return interfaces.toArray(new String[interfaces.size()]);
-   }
-
-   private void getInterfaces(Class<?> clazz, Set<String> interfaces) {
-      for (Class<?> itf : clazz.getInterfaces()) {
-         if (interfaces.add(itf.getName())) {
-            getInterfaces(itf, interfaces);
-         }
-      }
-      if (clazz.getSuperclass() != null) {
-         getInterfaces(clazz.getSuperclass(), interfaces);
-      }
-   }
-
-   public void unregister() {
-      if (registrations != null) {
-         for (ServiceRegistration<?> reg : registrations.values()) {
-            reg.unregister();
-         }
-      }
-   }
-}
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.activemq.artemis.osgi;
+

```

Comment:
You are mixing reformat changes here. It's hard to know what you changed.


Can you isolate only what was changed and make this a simple patch?



you can then:

```sh
git commit -a --amend
git push origin -f
```